### PR TITLE
fix(scheduler): improve job/task clone func

### PR DESCRIPTION
**Pull request by shinytang6**
Signed-off-by: shinytang6 <1074461480@qq.com>

### CODE REVIEW

File: pkg/scheduler/api/job_info.go [hunk 1]
```diff
@@ -190,6 +190,7 @@ func (ti *TaskInfo) Clone() *TaskInfo {
 		Name:           ti.Name,
 		Namespace:      ti.Namespace,
 		Priority:       ti.Priority,
+		PodVolumes:     ti.PodVolumes,
```
Comment (k82cn) [inline pkg/scheduler/api/job_info.go:193]: should we use deep-copy?

Comment (shinytang6) [reply pkg/scheduler/api/job_info.go:193]: We did not mutate PodVolumes directly, so l think it ok to not use deepcopy here

File: pkg/scheduler/api/job_info.go [hunk 2]
```diff
@@ -485,18 +486,20 @@ func (ji *JobInfo) Clone() *JobInfo {
...
 
-		TaskStatusIndex:  map[TaskStatus]tasksMap{},
-		TaskMinAvailable: ji.TaskMinAvailable,
-		Tasks:            tasksMap{},
-		Preemptable:      ji.Preemptable,
-		RevocableZone:    ji.RevocableZone,
-		Budget:           ji.Budget.Clone(),
+		TaskStatusIndex:       map[TaskStatus]tasksMap{},
+		TaskMinAvailable:      ji.TaskMinAvailable,
+		TaskMinAvailableTotal: ji.TaskMinAvailableTotal,
```
Comment (william-wang) [inline pkg/scheduler/api/job_info.go:498]: What does "TaskMinAvailableTotal" mean?

Comment (shinytang6) [reply pkg/scheduler/api/job_info.go:498]: the sum of task minAvailable

**Pull request merged**